CHANGELOG.md

```diff
@@ -20,6 +20,10 @@
   [Ornithologist Coder](https://github.com/ornithocoder)
   [#1093](https://github.com/realm/SwiftLint/issues/1093)
 
+* Make `joined_default_parameter` correctable.  
```

Comment:
there's no need to add an extra entry since the rule hasn't been released yet. Just update the previous one to say it's correctable 👍